content/en/docs/tutorials/clusters/seccomp.md

```diff
@@ -35,16 +36,63 @@ profiles that give only the necessary privileges to your container processes.
 
 ## {{% heading "prerequisites" %}}
 
+{{< version-check >}}
+
 In order to complete all steps in this tutorial, you must install
 [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) and
 [kubectl](/docs/tasks/tools/). This tutorial will show examples
-with both alpha (pre-v1.19) and generally available seccomp functionality, so
+both alpha (new in v1.22) and generally available seccomp functionality. You should
```

Comment:
nit: This tutorial shows examples of alpha (...) and generally available Seccomp functionality. You should ...

Reply:
nit: Check on capitalization of Seccomp (or seccomp).

Reply:
Fixed and switched to `seccomp`.